Rubberduck.Core/Common/ModuleExporter.cs

```diff
@@ -11,7 +11,8 @@ public class ModuleExporter : IModuleExporter
 
         public string ExportPath => TempFile
             ? ApplicationConstants.RUBBERDUCK_TEMP_PATH
-            : Path.GetDirectoryName(Assembly.GetAssembly(typeof(App)).Location);
+            // FIXME check that GetExecutingAssembly is similarly acceptable as .GetAssembly(typeof(App))
+            : Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
```

Comment:
If `App` is in the executing assembly, these two will be identical.

Reply:
The problem is: App is in `Rubberduck.Main` and this is not necessarily called from there, IIUC

Reply:
Is there a specific reason why `App` is in `Main` and not in `Core`?

Reply:
@MDoerner because it's COMVisible, and thusly has been moved to the COM entry point assembly. Main...